src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -58,23 +57,11 @@ export class AddBulkConcepts extends Component {
   }
 
   handleCielClick = (e) => {
-    this.setState({ otherSelected: false });
     this.sourceUrl = e.target.value;
     this.props.fetchSourceConcepts(this.sourceUrl);
     this.reset();
   };
 
-  otherSourceClick = () => {
-    this.setState({ otherSelected: true });
-    this.props.fetchConceptSources();
-    this.reset();
-  }
-
-  handleSourceSelect = (source) => {
-    this.sourceUrl = source.url.substring(1);
```

Comment:
How is this related to removing other?

Reply:
This was being used by the other search to search for the sources.